code/game/jobs/job/job_exp.dm

```diff
@@ -240,4 +240,18 @@ GLOBAL_PROTECT(exp_to_update)
 			"ckey" = "'[sanitizeSQL(ckey)]'",
 			"minutes" = jvalue)))
 		prefs.exp[jtype] += jvalue
-	addtimer(CALLBACK(GLOBAL_PROC, /proc/update_exp_db), 20, TIMER_OVERRIDE|TIMER_UNIQUE)
\ No newline at end of file
+	addtimer(CALLBACK(GLOBAL_PROC, /proc/update_exp_db), 20, TIMER_OVERRIDE|TIMER_UNIQUE)
+
+
+/proc/queen_age_check(client/C)
+	if(!C.prefs?.exp)
+		return FALSE
+	if(!CONFIG_GET(flag/use_exp_tracking))
+		return FALSE
+	if(CONFIG_GET(flag/use_exp_restrictions_admin_bypass) && check_other_rights(C, R_ADMIN, FALSE))
+		return FALSE
+	var/my_exp = C.prefs.exp[ROLE_XENOMORPH]
+	if(my_exp >= 60)
+		return FALSE
+	else
```

Comment:
I don't think it's much of an issue, but Ghommie and Spooky have been pointing at needless `else`.